app/grid.scss

```diff
@@ -1,133 +1,154 @@
 @font-face {
-  font-family: Fontin Regular;
-  src: url(fonts/fontin2_pc/Fontin-Regular.ttf);
+    font-family: Open_Sans;
+    src: url(fonts/Open_Sans/OpenSans-Regular.ttf);
 }
 
 @font-face {
-  font-family: Muli;
-  src: url(fonts/Muli/Muli-Regular.ttf);
+    font-family: Merriweather-Regular;
+    src: url(fonts/merriweather/Merriweather-Regular.ttf);
 }
 
 body {
-  margin: 0;
-  height: 100%;
-  width: 100%;
-  font-family: Arial, Roboto, sans-serif;
+    margin: 0;
+    height: 100%;
+    width: 100%;
+    font-family: Roboto;
 }
 
-h2 {
-  text-align: left;
-  color: rgb(45, 45, 45);
-  margin-bottom: 2px;
-  margin-top: 40px;
-  font-family: Roboto;
+#name {
+    font-size: 27px;
+    font-weight: bold;
+    margin-left: 10px;
+    padding-top: 20px;
 }
 
-.grid {
-  display: block;
+#title {
+    margin-left: 10px;
+    margin-top: 20px;
+    font-size: 13px;
+    font-weight: bold;
+    font-family: Roboto;
 }
 
-#main {
-  color: #fff;
-  font-size: 16px;
-  height: 100%;
-  width: 100%;
-  margin: 0;
+h4 {
+    text-align: left;
+    color: rgb(45, 45, 45);
+    margin-bottom: 20px;
+    margin-top: 40px;
+    font-family: Roboto;
```

Comment:
No need to specify the `font-family` here since it's already defined in `body`

Reply:
Thanks! Corrected!